stable/autonity-network/templates/deployments_observers.yaml

```diff
@@ -72,6 +70,7 @@ spec:
           - "--wsorigins"
           - "*"
           - "--metrics"
+          - "--pprof"
```

Comment:
You need to add as well 
``` 
- "--pprofaddr"
- "0.0.0.0"
```
If not pprof will only listen to loopback interface

Reply:
fixed